src/test/resources/net/sf/jabref/importer/fileformat/OvidImporterTestBib6.bib

```diff
@@ -1,6 +1,6 @@
 @Misc{,
   Title                    = {Correct positioning of percutaneous iliosacral screws with computer-navigated versus fluoroscopically guided surgery in traumatic pelvic ring fractures},
-  Author                   = {Verbeek, Jan MD and Hermans, Erik MD and van Vugt, Arie MD, PhD and Frolke, Jan Paul MD, PhD},
```

Comment:
Hmm, I'm wondering if it is possible to skip this expansion... Probably we will have to decide on either hardcoding MD (medical doctor) and disappoint people with Marie David as middle names or just live with it.